### PR TITLE
First react component - tabs

### PR DESCRIPTION
**Pull request by kwalker3690**
Hey @daverau-optimizely and @danoc, here's my first attempt at an oui react component, tabs! This PR doesn't have any tests in it yet, just wanted to get your guys feedback on this first pass! Let me know what you think

### CODE REVIEW

**Comment by danoc** [comment]
I'll take a look at this when I get back next Monday, but check out these links I have bookmarked that might be helpful:

* http://frend.co/components/tabs/
* https://cloudflare.github.io/cf-ui/#cf-component-tabs

(I just skimmed those FWIW – don't know how good they are.)

I'd also reach out to @jordangarcia and @jessicalc because we've talked about how to build tabs before.

File: src/components/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import classNames from 'classnames';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```
Comment (danoc) [inline src/components/Tabs/index.js]: Let's chat with Jordan about this. I think the preferred approach is to create only stateless components and he has some opinions on the best way to create tab components.

Comment (danoc) [inline src/components/Tabs/index.js]: He sent me this a while back FWIW:

```
(props) => {
  return (
    <TabContainer>
      <Tab isActive={props.activeTab === 'one'}>Tab one</Tab>
      <Tab isActive={props.activeTab === 'two'}>Tab two</Tab>
      <Tab isActive={props.activeTab === 'tree'}>Tab tree</Tab>
    </TabContainer>
  )
}
```

Comment (kwalker3690) [inline src/components/Tabs/index.js]: Oh interesting! Ok that seems like an easy change to make

**Comment by tommygiglio** [comment]
Just thought I'd creep on your work. Nice work, all. 👍 

-The Ghost of Tommy

**Comment by kwalker3690** [comment]
Closed because this is not a PR to devel

**Pull request closed without merging**